src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
```

Comment:
The `.split(';')` method correctly splits the string into individual style declarations based on the semicolon delimiter.